api/api/spec/spec.yaml

```diff
@@ -7648,8 +7656,6 @@ paths:
           $ref: '#/components/responses/UnauthorizedResponse'
         '403':
           $ref: '#/components/responses/PermissionDeniedResponse'
-        '404':
-          $ref: '#/components/responses/ResourceNotFoundResponse'
```

Comment:
The same as above, for all the previous existing 404 ResourceNotFoundResponse codes.

Reply:
I Reverted all the removed lines found in the `4.9.`0 branch like the following:
```
        '404':
          $ref: '#/components/responses/ResourceNotFoundResponse'
```